os/fs/smartfs/README.txt

```diff
@@ -1,90 +1,179 @@
-SMARTFS README
+jSmartFS README
 ^^^^^^^^^^^^^^
 
 This README file contains information about the implemenation of the TinyAra
 Sector Mapped Allocation for Really Tiny (SMART) FLASH file system, SMARTFS.
 
+The same has now been upgraded to jSmartFs with the addition of 2 key features -
+Filesystem Consistency and Two-Tier File Data Integrity.
+
 Contents:
 
-  Features
-  General operation
-  SMARTFS organization
-  Headers
-  Multiple mount points
-  SMARTFS Limitations
-  ioctls
-  Things to Do
+  > Features
+  > General operation
+  > SMARTFS organization
+  > Headers
+  > Multiple mount points
+  > SMARTFS Limitations
+  > ioctls
+  > Things to Do
 
 Features
 ========
 
   This implementation is a full-feature file system from the perspective of
-  file and directory access.  The SMART File System was designed specifically
+  file and directory access. The SMART File System was designed specifically
   for small SPI based FLASH parts (1-8 Mbyte for example), though this is not
-  a limitation.  It can certainly be used for any size FLASH and can work with
-  any MTD device by binding it with the SMART MTD layer and has been tested with
-  devices as large as 128MByte (using a 2048 byte sector size with 65534 sectors).
+  a limitation.  It can certainly be used for flash memories of any size and has
+  been tested with devices as large as 128MiB (using a 2048 byte sector size
+  with 65534 sectors).
   The FS includes support for:
+    - Support for multiple mount points on a single volume / partition.
     - Multiple open files from different threads.
-    - Open for read/write access with seek capability.
-    - Appending to end of files in either write, append or read/write
-      open modes.
-    - Directory support.
-    - Support for multiple mount points on a single volume / partition (see
-      details below).
-    - Selectable FLASH Wear leveling algorithym
-    - Selectable CRC-8 or CRC-16 error detection for sector data
+    - Directory Support
+    - Open for reading / writing with seek capability.
+    - Appending to the end of files and Overwriting existing file contents.
+    - Static Wear leveling algorithm.
+    - CRC-16 checksums for error detection in data
```

Comment:
we provide both of crc8 & 16 , you think 8 is unstable or some different opinion?

Reply:
I skipped 8 because journaling only works with 8.
However, let me add a line to clarify this.